setup.cfg

```diff
@@ -1,3 +1,73 @@
+[pytest]
```

Comment:
should this not be `[tool:pytest]` https://docs.pytest.org/en/stable/customize.html#setup-cfg as below

Reply:
You are correct, I've fixed it.

Reply:
There isanother section `[tool:pytest]` below